### ISSUE TITLE
Docking/window position should be preserved when closing and opening Terminal

### ISSUE THREAD

**Issue by UweKeim** [microsoft/terminal#1420]
**Steps to reproduce:**

1. Start Terminal.
1. Press <kbd>Win</kbd>+<kbd>Left</kbd> key to dock Terminal window to the left half of my monitor. The Terminal window starts at (0|0) left top position.
1. Exit Terminal.
1. Start Terminal.

**Actual results:**

After step 4, the Terminal window starts at some probably random (X|Y) position.

Now I have to repeat above step 2:

1. Press <kbd>Win</kbd>+<kbd>Left</kbd> key to dock the Terminal window to the left half of my monitor. 

This is really annoying. 

**Expected results:**

The Terminal window should start pixel perfect at the very same position where I closed it the last time. No offset or anything. Just start with the same **size** and the same **position**. No matter whether I manually resized the window or whether being docked before.

Google Chrome does this very well, Microsoft Edge based on Chromium does this very well, too, same does e.g. Visual Studio Code. I hope Terminal can do better, too.

**Additional information:**

I've described a [similar issue some years back over at Super User](https://superuser.com/q/659585/11720), together with some UI mockups.

**Comment by DHowett-MSFT** [comment]
Duplicate of #1387, #1043

**Comment by DHowett-MSFT** [comment]
Also, just pitching in: I disagree. I might open multiple windows and I absolutely don’t want them all spawning over top of each other.
